config/v1/0000_10_config-operator_01_authentication.crd.yaml

```diff
@@ -52,6 +52,10 @@ spec:
                 type:
                   description: type identifies the cluster managed, user facing authentication mode in use. Specifically, it manages the component that responds to login attempts. The default is IntegratedOAuth.
                   type: string
+                  enum:
```

Comment:
What happened in 4.14 if this string value had a different value for this field?

Reply:
we had a separate validation in our o/k fork that did not allow this - https://github.com/openshift/kubernetes/pull/1770/